src/editors/containers/ProblemEditor/components/EditProblemView/SettingsWidget/settingsComponents/SwitchEditorCard.jsx

```diff
@@ -60,6 +56,8 @@ const SwitchEditorCard = ({
 SwitchEditorCard.propTypes = {
   problemType: PropTypes.string.isRequired,
   editorType: PropTypes.string.isRequired,
+  // eslint-disable-next-line react/forbid-prop-types
+  editorRef: PropTypes.object.isRequired,
```

Comment:
This is fine, but I would have preferred we pass the editorRef around using a `ProblemEditorContext` instead of prop drilling.